play-2.4/swagger-play2/app/play/modules/swagger/routes/RoutesFileParser.scala

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package play.modules.swagger.routes
+
+import java.io.File
+
+
+import play.api.Logger
+import play.modules.swagger.SwaggerUtils
+import play.modules.swagger.routes.Route
+
+import scala.io.Codec
+import scala.language.postfixOps
+import scala.util.{Right, Left}
+import scala.util.parsing.combinator._
+import scala.util.parsing.input._
+
+
+/**
+  * backport form playframework compile
```

Comment:
form -> from

A better way to do this would be to add a dependency on com.typesafe.play : routes-compiler_2.10 : 2.4.3 so that you don't have to copy/paste the code here